openedx/core/djangoapps/credentials/models.py

```diff
@@ -1,3 +1,67 @@
 """
 Models for credentials support for the LMS and Studio.
 """
+
+from urlparse import urljoin
+
+from django.utils.translation import ugettext_lazy as _
+from django.db import models
+
+from config_models.models import ConfigurationModel
+
+
+class CredentialsApiConfig(ConfigurationModel):
+    """
+    Manages configuration for connecting to the Credential service and using its
+    API.
+    """
+    api_version_number = models.IntegerField(verbose_name=_("API Version"))
+    internal_service_url = models.URLField(verbose_name=_("Internal Service URL"))
+    public_service_url = models.URLField(verbose_name=_("Public Service URL"))
+
+    enable_learner_issuance = models.BooleanField(
+        verbose_name=_("Enable Learner Issuance"),
+        default=False,
+        help_text=_(
+            "Enable issuance of credentials via Credential Service."
+        )
+    )
+    enable_studio_authoring = models.BooleanField(
+        verbose_name=_("Enable Authoring of Credential in Studio"),
+        default=False,
+        help_text=_(
+            "Enable authoring of Credential Service credentials in Studio."
+        )
+    )
+
+    def __unicode__(self):
+        return self.api_version_number
+
+    @property
+    def internal_api_url(self):
+        """
+        Generate a URL based on internal service URL and API version number.
+        """
+        return urljoin(self.internal_service_url, '/api/v{}/'.format(self.api_version_number))
```

Comment:
Taken from the previous PR: A major version change for the API means parts of it are backward-incompatible. If that's the case, you will most likely need to change the code, so being able to configure the API version is worthless. Either hardcode the value (`/api/v1/`) or add a field to take the complete URL.